packages/data-broker/src/__tests__/services/dhis/DhisInputSchemeResolvingApiProxy.stubs.js

```diff
@@ -26,14 +27,51 @@ const createModelsStub = () => ({
   },
   dataServiceEntity: {
     find: async filter =>
-      DATA_SERVICE_ENTITIES.filter(mapping => filter.entity_code.includes(mapping.entity_code)),
+      filter[QUERY_CONJUNCTIONS.RAW]
+        ? DATA_SERVICE_ENTITIES.filter(mapping =>
+            filter[QUERY_CONJUNCTIONS.RAW].parameters.includes(mapping.config.dhis_id),
+          )
+        : DATA_SERVICE_ENTITIES.filter(mapping => filter.entity_code.includes(mapping.entity_code)),
   },
 });
 
 export const createApiStub = () => {
   return {
     getAnalytics: jest.fn().mockReturnValue({ SOME_ANALYTICS_RESPONSE: 1 }),
-    getEventAnalytics: jest.fn().mockReturnValue({ SOME_EVENT_ANALYTICS_RESPONSE: 1 }),
+    getEventAnalytics: jest.fn().mockImplementation(() => ({
+      headers: [
+        {
+          name: 'oucode',
+          column: 'Organisation unit code',
+          valueType: 'TEXT',
+          type: 'java.lang.String',
+          hidden: false,
+          meta: true,
+        },
+        {
+          name: 'ou',
+          column: 'Organisation unit',
+          valueType: 'TEXT',
+          type: 'java.lang.String',
+          hidden: false,
+          meta: true,
+        },
+        {
+          name: 'Dyq13cMGMzT',
+          column: 'NCLE: Disease name',
+          valueType: 'TEXT',
+          type: 'java.lang.String',
+          hidden: false,
+          meta: true,
+          optionSet: 'kMe7B54S9VH',
+        },
+      ],
+      rows: [['dhisCode_ou1-Not-Mapped', 'dhisId_ou1', '7.1']],
```

Comment:
I understand Laos doesn't have org unit codes - out of interest, what do we get back from DHIS2 in the `code` cell? Is it empty string, null, or the column is just excluded from the response?

Reply:
Actually, we do get a code returned (at least for the org units in the initial POC)! However, we have been told to not rely on this code, I'm not too sure why though.

Reply:
Oh...ok then! 🤷